components/FlexibleHeader/src/MDCFlexibleHeaderView.m

```diff
@@ -385,8 +389,11 @@ - (void)layoutSubviews {
 
   [self fhv_updateShadowColor];
   [self fhv_updateShadowPath];
+
   [CATransaction begin];
-  [CATransaction setDisableActions:YES];
+  BOOL allowCAActions = _isAnimatingTrackingScrollViewChange &&
+                        self.allowShadowLayerFrameAnimationsWhenChangingTrackingScrollView;
+  [CATransaction setDisableActions:!allowCAActions];
```

Comment:
just want to verify; setDisableActions will be NO only when opted-in AND is within an animate block. This is what you intended?

Reply:
Yep; gotta love double-negative APIs >.<